doc/reference/networking/lwm2m.rst

```diff
@@ -254,14 +254,26 @@ To use the LwM2M library, start by creating an LwM2M client context
 	/* LwM2M client context */
 	static struct lwm2m_ctx client;
 
-Create callback functions for LwM2M resources that you wish to have actions
-for:
+Create callback functions for LwM2M resources that you wish to have create or
+delete actions for:
 
 .. code-block:: c
 
-	static int device_reboot_cb(uint16_t obj_inst_id)
+	static int resource_create_cb(uint16_t obj_inst_id)
```

Comment:
This callback is not registered in this example code, I don't think there's need for it. Just please update the existing docs, we don't need to extend it in this PR.